src/miniscript/analyzable.rs

```diff
@@ -22,7 +22,115 @@ use core::fmt;
 use std::error;
 
 use crate::prelude::*;
-use crate::{Miniscript, MiniscriptKey, ScriptContext};
+use crate::{Miniscript, MiniscriptKey, ScriptContext, Terminal};
+
+/// Params for parsing miniscripts that either non-sane or non-specified(experimental) in the spec.
+/// Used as a parameter [`Miniscript::from_str_insane`] and [`Miniscript::parse_insane`].
```

Comment:
In 89169c23aec0bac4009593e6c451bf9b0432ee6d:

This comment is out of date since you renamed these two methods :)